pymatgen/io/lobster/inputs.py

```diff
@@ -335,7 +336,7 @@ def get_basis(
         potcar_symbols: list[str],
         address_basis_file: PathLike | None = None,
     ) -> list[str]:
-        """Get the basis from given potcar_symbols, e.g., ["Fe_pv", "Si"].
+        """Get the basis functions from given potcar_symbols, e.g., ["Fe_pv", "Si"].
```

Comment:
ah, great to see that you changed it here as well :D

Reply:
Yep :) thanks